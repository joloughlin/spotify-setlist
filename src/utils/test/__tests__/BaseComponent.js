import React from 'react';
import { shallow } from 'enzyme';
import { BaseComponent } from '../BaseComponent';

describe('BaseComponent', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<BaseComponent />);
  });

  it('renders', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
