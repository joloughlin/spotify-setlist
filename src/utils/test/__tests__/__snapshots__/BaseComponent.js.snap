// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseComponent renders 1`] = `"<div></div>"`;
